### PR TITLE
Reacts redesign icon palette

### PR DESCRIPTION
**Pull request by Raemon**
Reworks the React Palette to 
– showcase the most common/easily-understood reacts at the top, to fit more on the component at once
– adds "likelihood" reacts, so people can flag their specific epistemic disagreements
– restructures both the ListView and GridView to take in a custom ordering of reacts, so that it's easier to make sure paired-reacts are next to each other.
– adds some infrastructure for custom-styling react icons
– adds a bunch more react icons and renames reacts/descriptions

image

image

image

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204811529546224) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/voting/reactions.tsx [hunk 1]
```diff
@@ -74,16 +77,16 @@ export const namesAttachedReactions: NamesAttachedReactionType[] = [
   {
     name: "agree",
     label: "Agreed",
-    searchTerms: ["check", "correct"],
+    searchTerms: ["check", "correct", "confirm", "upvote", "+1"],
     svg: "/reactionImages/nounproject/check.svg",
-    description: "I agree with this.",
+    description: "This seems true to me.",
```
Comment (darkruby501) [inline packages/lesswrong/lib/voting/reactions.tsx:82]: I actually think we should remove descriptions on the more obvious ones.

File: packages/lesswrong/lib/voting/reactions.tsx [hunk 2]
```diff
@@ -200,14 +212,14 @@ export const namesAttachedReactions: NamesAttachedReactionType[] = [
...
-    filter: {opacity: 0.6},
+    svg: "/reactionImages/nounproject/noun-mail-checkmark-5316519.svg",
+    filter: {opacity: 0.6, scale: 1.2, translateY: 2},
     description: "I covered this in my post and/or comments.",
     deprecated:false,
   },
   {
     name: "unnecessarily-combative",
-    label: "Unnecessarily Combative",
+    label: "Too Combative?",
```
Comment (darkruby501) [inline packages/lesswrong/lib/voting/reactions.tsx:222]: I think the "unnecessarily" does important work in not saying that combativeness is not generically something we are against.

File: packages/lesswrong/components/votes/ReactionsPalette.tsx [hunk 3]
```diff
@@ -158,17 +191,81 @@ const ReactionsPalette = ({getCurrentUserReaction, getCurrentUserReactionVote, t
...
         <ReactionDescription reaction={reaction} classes={classes}/>
       </div>
-    </Row>
+    </div>
   }
+
+  const getReactionFromName = (name: string) => namesAttachedReactions.find(r => r.name === name && reactionsToShow.includes(r));
+
+  const primary = [
+    'agree',      'disagree',   'important',    'dontUnderstand', 'shrug', 'thinking',   'surprise',   'roll', 'confused',
```
Comment (darkruby501) [inline packages/lesswrong/components/votes/ReactionsPalette.tsx:208]: What's with the weird spacing?

**Review by darkruby501 — APPROVED** [review]

**Comment by Raemon** [comment]
Here's the final palette:

Before show-more:
image

After show more:
image

**Pull request merged**